src/include/gsh_config.h

```diff
@@ -145,7 +145,7 @@ typedef enum protos {
 /**
  * @brief Default value for core_param.rpc.debug_flags
  */
-#define TIRPC_DEBUG_FLAGS 0x0
+#define TIRPC_DEBUG_FLAGS TIRPC_DEBUG_FLAG_ERROR
```

Comment:
At a high-level looks good to me.
Basically test the following scenarios :
1. If libntirpc errors are getting logged by default.
2. If libntirpc debugs can be enabled through config file or dbus ?